app/models/issue.rb

```diff
@@ -18,6 +19,18 @@ def get_interfaces(interface = nil)
   end
 
   def stacktrace_frames
+    #added a switch statement in case errors from different platforms will be saved differenty
+    case platform
+    when 'javascript'
+      frames = get_interfaces(:stacktrace)._data[:frames]
+      frames = 'Missing stacktrace' if frames.blank?
```

Comment:
see that `get_platform_frames` is equal to `[]` not a message. So use the same return here don't try to return a message for javascript but for other platforms return an empty array. Work with the same value so we can build code that behaves the same for all platforms.